blog/_posts/2018-07-13-team-akaringular.md

```diff
@@ -0,0 +1,36 @@
+---
+title: New Tech Journey With RGSoC!
+layout:post
```

Comment:
Hi @StlMaris123 ! You need to add one blank space between ':' and text here and in other parameters below.